lib/css_parser/rule_set.rb

```diff
@@ -154,7 +168,7 @@ def expand_shorthand!
     def expand_background_shorthand! # :nodoc:
       return unless @declarations.key?('background')
 
-      value = @declarations['background'][:value]
+      value = @declarations['background'].value.dup
```

Comment:
can we do the `dup` only whenever we are about to modify to avoid extra work/strings ?